src/Kris/LaravelFormBuilder/FormBuilderServiceProvider.php

```diff
@@ -30,7 +30,7 @@ public function register()
 
         $this->app->singleton('laravel-form-builder', function ($app) {
 
-            return new FormBuilder($app, $app['laravel-form-helper']);
+            return new FormBuilder($app, $app['laravel-form-helper'], $app['Illuminate\Contracts\Events\Dispatcher']);
```

Comment:
I think `'Illuminate\Contracts\Events\Dispatcher'` can be simply replaced with `'events'`.

https://github.com/laravel/framework/blob/5.2/src/Illuminate/Foundation/Application.php#L1074